### PR TITLE
Complete Seeding Paper Code

### PR DESCRIPTION
**Pull request by wangpatrick57**
Added all settings to BLANT needed to generate index, as well as entire Python code for seeding algorithm. Updated USAGE_MSG for new BLANT settings and README.md for index creation algorithm and local alignment algorithm.

### CODE REVIEW

**Review by waynebhayes — CHANGES_REQUESTED** [review]
There's a few files that should not be in the repo. Please remove them.
- .firsttime is created once the user runs a successful "make", but it should not be in the repo. Maybe also add it to ".gitignore" :-)
- .pyc files are compiled Python... did you intend to submit those? Binary files are generally a bad idea in a repo.
- syeast has a bunch of .orca4 files. I'm guessing those are OK but please confirm (shouldn't require the user to have ORCA.... or maybe we can make it a sub-repo?
- libcalci.so also
- basically do a "make pristine" before you commit
- git-at shouldn't be in the repo
- patching_oldBlant files?
- regression-tests/recursiveDijkstraTest/seed7 is created during regression

I'm only 5% of the way down the list of commits. No way all those files should be part of it. Reduce the file count to the absolute minimum and try again.
